### PR TITLE
added support for commas in env and labels #555

### PR DESCRIPTION
**Pull request by alexbumbacea**

### CODE REVIEW

**Comment by alexbumbacea** [comment]
I signed it!

**Comment by alexbumbacea** [comment]
hi @nlopezgi,
Added a test case for csv in env

File: tests/docker/BUILD [hunk 1]
```diff
@@ -176,3 +176,18 @@ file_test(
...
+container_image(
+    name = "set_env_csv",
+    base = "@distroless_fixed_id//image",
+    cmd = ["foo"],
+)
+
+container_test(
+    name = "set_env_csv_test",
+    configs = ["//tests/docker/configs:set_env_csv.yaml"],
+    env = {
```
Comment (nlopezgi) [inline tests/docker/BUILD]: container test does not allow env attr, it only receives the configs to validate with

Comment (nlopezgi) [reply tests/docker/BUILD]: (the env should be set in the container_image above)

**Comment by nlopezgi** [comment]
also //container:image_test is failing, you should be able to run that locally and debug

**Comment by nlopezgi** [comment]
ugh, mac builds are timing out pulling images, will have to figure out tomorrow what the problem is

**Pull request merged**